variables.tf

```diff
@@ -109,6 +112,18 @@ variable "users" {
     ]))
     error_message = "group type must be either 'USER', 'GROUP', or 'CUSTOMER'"
   }
+
+  # validate that schema_values's values can be JSON encoded (required by Google Workspace provider)
+  validation {
+    condition = alltrue(flatten([
+      for user in var.users : [
+        for schema in user.custom_schemas : [
+          for key, value in schema.schema_values : can(jsonencode(value))
+        ]
+      ]
+    ]))
+    error_message = "All values in custom schema values must be JSON encodable strings"
```

Comment:
Ah that's why we have weird strings. Okay. Good to know -- Glad we have this validation as documentation.